packages/account/src/Components/poi/status/unsupported/card-details/card-details.tsx

```diff
@@ -9,7 +9,7 @@ type TCardDetails = {
     data: FormikValues;
     onComplete: (e: object) => void;
     goToCards: () => void;
-    is_from_external: boolean;
+    is_from_external?: boolean;
     setIsCfdPoiCompleted: (e: boolean) => void;
```

Comment:
on my opinion, here param 'e' looks a little bit confusing, because usually we use such name as a shortening of 'event'.
could you please rename it :)